spanner/spanner_snippets/spanner/integration_test.go

```diff
@@ -1332,6 +1332,44 @@ func TestAddSplitPointsSample(t *testing.T) {
 	assertContains(t, out, "Added split points")
 }
 
+func TestDmlWithLastStatementSample(t *testing.T) {
+	_ = testutil.SystemTest(t)
+	t.Parallel()
```

Comment:
issue: don't run these tests in parallel unless you are explicitly testing parallelization. This can cause race conditions and/or resource contention in our CI/CD.